src/node-types.js

```diff
@@ -0,0 +1,9 @@
+/**
+ * Created on 8/17/16.
+ */
+const NODE_TYPES = {
```

Comment:
I saw that you use `defs`, `link` and `node` as a separate variables above. If you do that you should consider using the same constant there as well